airflow/www/templates/airflow/dag.html

```diff
@@ -21,6 +21,10 @@
 
 {% block page_title %}{{ dag.dag_id }} - Airflow{% endblock %}
 
+{% block head_meta %}
+  <meta name="robots" content="noindex" />
```

Comment:
!! This should call `{{ super() }}`

Reply:
@ashb  sorry can you be a bit clearer / detailed.

Reply:
Look at the next block 4 lines down